docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -613,6 +613,21 @@ oc logs -l app=openstackansibleee -f --max-log-requests 20
 oc wait --for condition=Ready openstackdataplanedeployment/openstack-pre-adoption --timeout=10m
 ----
 
+. If the validation deployment failed then check the ansible execution logs to
```

Comment:
if any openstack-pre-adoption validations fail, you must first determine which ones were unsuccessful and then follow the instructions below for each case:

(just a suggestion feel free to adopt part/all/none)

Reply:
applied your suggestion with a small tweak